### PR TITLE
feat:creating call to action section

### PR DESCRIPTION
**Pull request by arjorb**
## Describe your changes
we created a call to action section and add a button component to it for scheduling a call
## Issue ticket number and link
id -> 017
link -> [Here](https://www.notion.so/apeunit/call-to-action-section-8e171fa6ea424fb78bf7afb5572622ba)
## Tasks completed
- [x] Creating a call to action section
## Tasks not completed
None
## Screenshots (if needed)
None

### CODE REVIEW

File: components/sections/homepage/CallToAction/index.jsx [hunk 1]
```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import Wrapper from '../../../layouts/Wrapper';
+import Button from '../../../ui/Button';
+
+const CallToAction = () => {
+  return (
+    <Wrapper size='small'>
+      <div className='hidden lg:block w-full h-max bg-gray-8 rounded-4xl py-16 text-center mt-32'>
+        <h1 className='text-custom-3xl'>Ready to talk about your project?</h1>
```
Comment (jacob-js) [inline components/sections/homepage/CallToAction/index.jsx:9]: This heading must have a font weight of 600

File: components/sections/homepage/CallToAction/index.jsx [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+        <p className='text-center mx-auto text-custom-md mt-5 text-gray-1'>
+          Are you looking for a consultancy partnership to address your blockchain challenges? <br /> We’d love to hear
+          from you. Let’s build something <br /> amazing together.
+        </p>
```
Comment (jacob-js) [inline components/sections/homepage/CallToAction/index.jsx:13]: This paragraph here doesn't have any margin top, make sure to check the value correctly in Figma

File: components/sections/homepage/CallToAction/index.jsx [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+        <div className='text-black w-full flex justify-center pt-16 px-5'>
+          <Button>Schedule a call</Button>
```
Comment (jacob-js) [inline components/sections/homepage/CallToAction/index.jsx:15]: Make sure to use the button's `size` prop to make it a bit large than it is. The value `large` it is used to give it a large size

**Review by jacob-js — CHANGES_REQUESTED** [review]
# Good job @arjorb 

Only some few issues to fix. kindly refer to my comments

**Review by mutesialine — DISMISSED** [review]
It's looks good

**Review by jacob-js — APPROVED** [review]

**Review by mutesialine — APPROVED** [review]

**Pull request merged**